README.rst

```diff
@@ -671,3 +671,45 @@ To include a ``"foo.cfg"`` config file, use the following statement::
     include("foo.cfg")
 
 This allows to re-use, for example flash configuration files, across different image configurations.
+
+License and Developing
+======================
+
+To contribute to umpf please prepare a pull request on Github. To make
```

Comment:
Should I guess where you copied that text from? :-)

Reply:
Fixed.